builder/vmware/common/driver.go

```diff
@@ -642,8 +647,46 @@ func GetOVFTool() string {
 	return ovftool
 }
 
+func CheckOvfToolVersion(ovftoolPath string) bool {
+	output, err := exec.Command(ovftoolPath, "--version").CombinedOutput()
+	if err != nil {
+		return false
+	}
+	versionOutput := string(output)
+
+	versionRegex, err := regexp.Compile(`\d+\.\d+\.\d+`)
```

Comment:
FYI, this could be a global variable, coupled with `regexp.MustCompile`, so that you immediately know if the regex is invalid at plugin boot.

Ex:
```go
var ovfToolVersionRegex = regexp.MustCompile(`\d+\.\d+\.\d+`)
```

Reply:
Resolved in 939a252.